app/bitcoin/servers.js

```diff
@@ -1,29 +1,78 @@
 class BitcoinServerRequests {
 
     getBalance(address) {
-        const response = this.makeRequest(`https://www.blockchain.info/address/${address}?format=json`);
-        if (response) {
-            const json = JSON.parse(response.body);
-            return json.final_balance;
-        }
-        return -1;
+        const promise = this._getRequest('GET', `https://www.blockchain.info/address/${address}?format=json`);
+
+        promise.then(
+            (val) => {
+                const json = JSON.parse(val.body);
```

Comment:
I'd recommend putting the callback definition on the same line as the function being called, e.g.:
```
promise.then((val) => {
```
or
```
.catch(() => -1)